controllers/host/host_controller.go

```diff
@@ -1172,19 +1172,6 @@ func (r *HostReconciler) ReconcileHostByState(
 			return r.CloudManager.StartMonitor(m, msg)
 		}
 
-		// Clean up strategy required after Disabled and enabled attributes are all in-sync
-		if strategy_required {
```

Comment:
The strategy not required is set intentionally if !CompareDisabledAttributes. Removing this setion may cause the host stuck in other status(unlock_required) by unpredicted errors(strategy abort etc), this section should not be removed. In case you don't want to set it not required by other condition, please add it above